### ISSUE TITLE
OpenWrt Logo spacing requirements 

### ISSUE THREAD

**Issue by richb-hanover** [aparcar/openwrt-branding#2]
**TL;DR:** General advice - just add spacing of 20% of the image HEIGHT to top, bottom, left, right. That is, if the image is 500 pixels high, use 100px (20%) on all four sides of the image to give it "enough breathing room".

**The details:** On Page 3 of the OpenWrt Style Guide (https://github.com/openwrt/branding/blob/master/openwrt-styleguide.pdf), the requirements for spacing are impossible to determine because the "X" height in each of the images is not referenced to any fixed value (see image below):

1. In the Logo-above-OpenWrt, the X-height is the height of the logo itself, which has no externally-determined value
2. In the Wordmark (text-only), the X-height is the text base line, measured to the top of the ® symbol, however that is not referenced to any point
3. In the Logo-beside-OpenWrt, the X-height is set from the baseline of the text to the top of the logo - again, without any reference point
4. In the Logo-alone case, the X-height _might_ be the same as the width of the logo, since the "O" appears to be a circle.

To get the 20% approximation above, I opened the images (below) in my graphics program and measured:

1. Logo-above-OpenWrt: Outline of image is 3060w x 2460h pixels; X-height is 1657 pixels. Therefore X-height is 1657/2460 or 67.3% of the height of the image; spacing is 1/4 of that, or 16.8% of the image height

2. Wordmark: Outline of image is 5877w x 1481h pixels; X-height is 1141 pixels. Therefore X-height is 1141/1481, or 77.0% of the height of the image; spacing is 1/4 of that or 19.2% of the image height.

3. Logo-beside-OpenWrt: Outline of image is 4441w x 1353h pixels; X-height is 1164 pixels. Therefore X-height is 1164/1353, or 86.0% of the height of the image; spacing is 1/4 of that or 21.5% of the image height.
	
4. Logo-alone: Outline of image is 1137w x 1365h pixels; X-height is 1120 pixels. Therefore X-height is 1120/1365, or 82.1% of the height of the image; spacing is 1/4 of that or 20.5% of the image height.

![image](https://github.com/openwrt/branding/assets/1094930/b686098c-31b5-4353-805c-690b413dc887)
